js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -122,15 +122,15 @@ applyToResourceTypeFlag=Apply to resource type
 offlineSessionIdleHelp=Time an offline session is allowed to be idle before it expires. You need to use offline token to refresh at least once within this period; otherwise offline session will expire.
 eventTypes.UPDATE_TOTP.description=Update totp
 testError=Error when trying to connect to LDAP\: '{{error}}'
-groupObjectClassesHelp=Object class (or classes) of the group object. It's divided by commas if more classes needed. In typical LDAP deployment it could be 'groupOfNames'. In Active Directory it's usually 'group'.
+groupObjectClassesHelp=Object class (or classes) of the group object. It's divided by commas if more classes needed. In typical LDAP deployment it could be 'groupOfNames'. In Active Directory it is usually 'group'.
```

Comment:
```suggestion
groupObjectClassesHelp=Object class (or classes) of the group object. It is divided by commas if more classes are needed. In a typical LDAP deployment, it could be 'groupOfNames'. In Active Directory, it is usually 'group'.
```